source/jwtd/jwt.d

```diff
@@ -119,7 +120,11 @@ JSONValue decode(string token, string delegate(ref JSONValue jose) lazyKey) {
 	JWTAlgorithm alg;
 	try {
 		// toUpper for none
-		alg = to!(JWTAlgorithm)(toUpper(header["alg"].str()));
+		if (toUpper(header["alg"].str) == "EDDSA") {
```

Comment:
nit: can consider running `toUpper` once instead of twice for the non-EDDSA case